sdks/go/pkg/beam/core/runtime/graphx/coder_test.go

```diff
@@ -71,8 +71,8 @@ func TestMarshalUnmarshalCoders(t *testing.T) {
 			coder.NewWKV([]*coder.Coder{foo, bar}, window.NewGlobalWindow()),
 		},
 		{
-			"W<GBK<foo,bar>>",
-			coder.NewWGBK([]*coder.Coder{foo, bar}, window.NewGlobalWindow()),
+			"W<CoGBK<foo,bar>>",
+			coder.NewWCoGBK([]*coder.Coder{foo, bar}, window.NewGlobalWindow()),
```

Comment:
<!--new_thread; commit:4c15f6b089ab3ba3d41a887816755a22f6be7e5e; resolved:0-->
Does it make sense to add a CoGBK for more than the "identical to the GBK" case? eg. with foo, bar, baz?